tests/basic/misc_test.go

```diff
@@ -1784,4 +1785,58 @@ var _ = Describe("{KubeClusterRestart}", func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
-})
\ No newline at end of file
+})
+
+var _ = Describe("{VerifyNoPxRestartDueToPxPodStop}", func() {
+	JustBeforeEach(func() {
+		var testrailID = 87311294
+		StartTorpedoTest("VerifyNoPxRestartDueToPxPodStop", "Verify that px serivce remain up even if px pod got deleted ", nil, 0)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	It("has to setup, validate and teardown apps", func() {
+
+		// Get uptime for px service on each node
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			processPid := make(map[string]string)
+			startCmd := "pidof px"
+			//Capturing PID pf PX before stopping PX pods
+			for _, nnode := range node.GetStorageNodes() {
+				output, _ := Inst().N.RunCommand(nnode, startCmd, node.ConnectionOpts{
+					Timeout:         30 * time.Second,
+					TimeBeforeRetry: 20 * time.Second,
+					Sudo:            true,
+				})
+				processPid[nnode.Id] = output
+			}
+			log.Infof(fmt.Sprintf("Process IDs for px before stopping portworx pod  %s", processPid))
+
+			//Deleting px pods from all the node
+			err = DeletePXPods("kube-system")
+			if err != nil {
```

Comment:
no need for if check here, FailOnError will take care of the nil check

Reply:
Removed

Reply:
How about?

```go
log.FailOnError(err, "unable to get portworx namespace")
```

1. No need of wrapping `err` with `fmt.Errorf`.
2. In case of an error, the namespace will be empty, so there might not be any use in including it in the log.